### PR TITLE
Connect constraint form to backend

### PR DESCRIPTION
**Pull request by bdeak4**
This PR connects constraint form to backend and adds 'Insert seed' button to constraint form to make testing easier.

### CODE REVIEW

File: backend/src/routes/timetable.ts [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+    classes: classes.map(classItem => ({
+      ...classItem,
+      _id: id++,
+      subjects: classItem.subjects.map(subject => ({
+        ...subject,
+        _id: id++,
+        classroom: subject.classroom || { name: '' },
+      })),
+    })),
+  };
```
Comment (abasic) [inline backend/src/routes/timetable.ts]: Niptick:
```js
function subjectsMapper = (subject, id) => {
  const classroom = subject.classroom || { name: '' };
  return { ...subject, _id: id, classroom };
};

const classes = classes.map(classItem => {
  const _id = id++;
  const subjects = classItem.subjects.map(subject => subjectsMapper(subject, id++));
  return{ ...classItem, _id, subjects};
});
```

**Review by abasic — DISMISSED** [review]

File: frontend/src/api/timetable.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import fetcher from './helpers';
 
 const baseUrl = '/api/timetable';
 
-const getTimetable = () => fetcher(`${baseUrl}/`);
+const getTimetable = formData => fetcher(`${baseUrl}/`, { method: 'POST', body: formData });
```
Comment (nkelava) [inline frontend/src/api/timetable.js]: Nitpick: if you could write this in multiple lines.

File: frontend/src/components/ConstraintForm/ConstraintForm.vue [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
-    const generateTimetable = () => {
-      // TODO: submit handler will be implemented when algorithm backend is merged
-      console.log(formStore.form);
+    const handleSubmit = () => {
+      router.push({ name: 'timetables' });
     };
 
-    return { formStore, generateTimetable };
+    const insertSeed = () => {
+      timetableService.getSeed().then(json => (formStore.form = json));
```
Comment (nkelava) [inline frontend/src/components/ConstraintForm/ConstraintForm.vue:18]: Nitpick: same as above.

**Review by nkelava — DISMISSED** [review]
Looks good! 🚀 Only some nitpicks that are personal preferences 👍

File: frontend/src/components/ConstraintForm/ConstraintForm.vue (hunk 3 above)
Comment (bdeak4) [reply frontend/src/components/ConstraintForm/ConstraintForm.vue:18]: prettier does not let me do this

**Review by nkelava — APPROVED** [review]

**Pull request merged**